mvp-spec/partials-updater/package-lock.json

```diff
@@ -1,4572 +1,8 @@
 {
   "name": "partials-updater",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
   "requires": true,
-  "packages": {
```

Comment:
Should we really commit this change to package-lock.json ?

Reply:
Actually there were still crto urls in the package-lock.json file so it's good we fix it ;-)
https://github.com/prebid/addressability-framework/pull/126